src/stories/layouts/Wizard/Wizard.js

```diff
@@ -0,0 +1,106 @@
+import './styles.scss';
+
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import StepWizard from 'react-step-wizard';
+import { Steps } from '../../components/Steps/Steps';
+import { Constrain } from '../../components/Constrain/Constrain';
+
+/**
+ * Component for Wizard layout element.
+ * @param {node} children of the component
+ * @param {array} labels of steps
+ * @return {object} (
+ *   <Wizard labels={labels}>
+ *       {children}
+ *   <Wizard />
+ * )
+ */
+/* eslint react/prop-types: 0 */
+export const Wizard = ({children, labels}) => {
+    const [state, updateState] = useState({
+        form: {},
+    });
+
+    // Do something on step change
+    const onStepChange = (stats) => {
+        // might need later on
+    };
+
+    const setInstance = (SW) => updateState({
+        ...state,
+        SW,
+    });
+
+    const { SW } = state;
```

Comment:
Can we change this to a more descriptive name?

Reply:
Yes!